src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final byte MAX_RND_NUMBER_FIGURE = 5;
+    private static final byte MAX_RND_NUMBER_100 = 100;
+    private static final byte RADIUS_WHITE_CIRCLE = 10;
+    private static Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle("WHITE", RADIUS_WHITE_CIRCLE);
+    }
+
+    private int getRandomInt() {
+        return new Random().nextInt(MAX_RND_NUMBER_100);
+    }
+
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(MAX_RND_NUMBER_FIGURE);
+        switch (index) {
+            case 0:
+                return new Circle(colorSupplier.getRandomColor(), getRandomInt());
```

Comment:
```suggestion
            case 0:
                return getRandomCircle();
```
lets make a separate method that creates a random circle, make similar methods for each figure